pkg/stripe/client.go

```diff
@@ -29,6 +29,10 @@ type Client struct {
 	// empty, the `Authorization` header will be omitted.
 	APIKey string
 
+	// When this is enabled, request and response headers will be printed to
+	// stdout.
+	Verbose bool
```

Comment:
Just FYI that once/if you add linting, it'll complain about stuff like this. Generally comments on exported variables/functions are prefixed with the name of the variable/function. e.g. "Verbose enables verbose mode in the client, which causes it to print requests and responses to stdout."

Reply:
We do have run both `go vet` and `golint`, but neither seem to complain here. Also the Go plugin for VSCode does show inline documentation for structs, but not for struct fields. I wonder if that's related.

Reply:
Oh weird! Okay, I'm probably misremembering then — maybe it's just functions and globals.